detox/ios/Detox/DTXMethodInvocation.m

```diff
@@ -15,58 +15,58 @@ @implementation DTXMethodInvocation
 
 + (id)getTarget:(id)param onError:(void (^)(NSString*))onError
 {
-    if (param == nil) return nil;
-    if ([param isKindOfClass:[NSDictionary class]])
-    {
-        NSDictionary *p = (NSDictionary*)param;
-        NSString *type = [DTXMethodInvocation getString:[p objectForKey:@"type"]];
-        id value = [p objectForKey:@"value"];
-        return [DTXMethodInvocation getValue:value withType:type onError:onError];
-    }
-    return nil;
+	if (param == nil) return nil;
+	if ([param isKindOfClass:[NSDictionary class]])
+	{
+		NSDictionary *p = (NSDictionary*)param;
+		NSString *type = [DTXMethodInvocation getString:[p objectForKey:@"type"]];
+		id value = [p objectForKey:@"value"];
+		return [DTXMethodInvocation getValue:value withType:type onError:onError];
+	}
+	return nil;
```

Comment:
nu ma :-/

Reply:
?

Reply:
spaces => tabs, much pointless diff

Reply:
It's possible to configure this in the diff view:

![whitespace](https://user-images.githubusercontent.com/2270433/60144926-fcaf9a00-97cc-11e9-98d6-e57165897353.png)